test/test_um2netcdf.py

```diff
@@ -651,6 +662,379 @@ def test_fix_units_do_nothing_no_um_units(ua_plev_cube):
         assert ua_plev_cube.units == orig  # nothing should happen as there's no cube.units
 
 
+@pytest.fixture
+def lat_river():
```

Comment:
For clarity, should the fixture name be tweaked to `lat_river_coord(s?)`?

Reply:
Good idea, I've updated all the coordinate fixture names.